SokolTextGame/Commands.cs

```diff
@@ -17,6 +17,7 @@ public class Exit : ICommand
     {
         public void Execute(string[] words, World world)
         {
+            Console.ForegroundColor = ConsoleColor.Blue;
             Console.WriteLine("See you later..");
```

Comment:
You changed the font color but forgot to reset it after `Console.WriteLine()`. This leads to a bug where if you exit the game and continue working in the same terminal window, all font color is now blue. Please add `Console.ResetColor();` after the `WriteLine`.